website/docs/guides/creating-a-new-package.md

```diff
@@ -22,6 +22,7 @@ Let's take a closer look at `Scarb.toml`:
 [package]
 name = "hello_world"
 version = "0.1.0"
+edition = "2023_01"
```

Comment:
```suggestion
edition = "2023_10"
```

Reply:
Actually, 01 is default now ![image](https://github.com/software-mansion/scarb/assets/18600023/3b3b1c7f-97f0-4127-a1cd-7181c85c3080)

Reply:
`01` is default, because that's the edition you get when you omit the `edition` field.

we want to push people to start new projects with `10`, see Cairo rel notes :)